src/main/resources/templates/start.html

```diff
@@ -1,59 +1,118 @@
+            <span class="navbar-toggler-icon"></span>
+        </button>
+        <div class="collapse navbar-collapse" id="navbarText">
+            <ul class="navbar-nav mr-auto">
+                <li class="nav-item active">
+                    <a class="nav-link" href="#">Home <span class="sr-only">(current)</span></a>
+                </li>
+                <!--TODO : Add navigation to other site pages-->
+                <!--Leaving these here so the developers can implement them later-->
+                <!--<li class="nav-item">
+                    <a class="nav-link" href="#">Features</a>
+                </li>
+                <li class="nav-item">
+                    <a class="nav-link" href="#">Pricing</a>
+                </li>-->
+            </ul>
+            <span class="navbar-text">Your ultimate source for finding and managing books</span>
         </div>
-        <div id="search-bar">
-            <input type="text" id="book-search" placeholder="Search for a book..." style="width: 300px;">
-            <button id="search-button">Search</button>
+    </nav>
+
+    <!--Search Bar-->
+    <br>
+    <div class="input-group mb-3">
+        <input type="text" class="form-control" placeholder="Search for a book..." aria-label="Search" aria-describedby="basic-addon2">
+        <div class="input-group-append">
+            <button class="btn btn-outline-secondary" type="button">Button</button>
         </div>
-    </header>
+    </div>
 
     <section id="homepage">
-        <div id="recent-arrivals">
-            <h2>Recent Arrivals</h2>
-            <!-- Recent arrivals content -->
-            <div class="book">
-                <img src="src/main/resources/templates/images/SpareBook.png" alt="Book 1">
-                <h3>Spare</h3>
-                <p>Author: Prince Harry the Duke of Sussex</p>
-                <p>Published: January 2023</p>
-                <p>Description: Spare is a memoir by Prince Harry. The book is Harry's account of his life to date. </p>
+        <hr>
+        <div class="container">
+            <div class="page-header text-center">
+                <h2>Recent Arrivals</h2>
             </div>
-            <div class="book">
-                <img src="src/main/resources/templates/images/Hangman.png" alt="Book 2">
-                <h3>Hangman</h3>
-                <p>Author: Maya Binyam</p>
-                <p>Published: August 2023</p>
-                <p>Description: Hangman is about exile, diaspora, and the impossibility of Black refuge in America and beyond.</p>
+        </div>
+            <hr>
+        <div class="container">
+            <div id="recent-arrivals" class="row">
+
+                <!-- Recent arrivals content -->
+                <div class="col">
+                    <figure class="figure">
+                        <img th:src="@{/images/SpareBook.png}" alt="Book 1" style="max-height: 724px; max-width: 472px">
```

Comment:
Fixed image display issue using Thymeleaf and spring boot by adding correct tag and adjusting image folder to reside in a static resource container